cmd/ip.go

```diff
@@ -136,6 +141,8 @@ func (c *ipCmd) Create(rq *ipAllocateRequest) (*models.V1IPResponse, error) {
 		return resp.Payload, nil
 	}
 
+	// FIXME this is a workaround, if specificIP is set, AF must not be set.
```

Comment:
What's the way that would not be a workaround?

Reply:
I think the actual implementation is the correct way, i remove the FIXME